tests/python/relax/test_transform_cse.py

```diff
@@ -90,6 +88,12 @@ def foo() -> R.Tuple(R.Tensor((), dtype="int32"), R.Tensor((2, 2), dtype="int32"
 
 
 def test_repeated_inner_tuples():
+    """CSE is only applied at variable bindings
+
+    To remain consistent with the behavior of the normalizer, tuples
+    are kept as-is, even if they contain repeated sub-tuples.
+    """
```

Comment:
Good observation, probably a good choice.

Reply:
Thank you.  With how much discussion there was around the normalization with respect to `R.call_tir`, I figured it's best to match as much as possible to the normalization.